main.tex

```diff
@@ -1524,7 +1512,7 @@ \subsection{Liquidity Staking Rewards}\label{liquidity-staking-rewards}}
 
 is calculated based on the staker's share of total iAsset staked:
 
-\[r=k\frac{xy/z}{b\sum_{i=1}^{\left| a \right|}a_{i}b_{i}/c_{i}}\]
+\[r=k\frac{x}{\sum_{i=1}^{\left| a \right|}a_{i}}\]
```

Comment:
There's still some confusion left in the formula because one iAsset can have multiple corresponding LP tokens.

Suppose `iUSD` iAsset has 2 LP tokens -- `lp1` & `lp2`.

`k` was previously defined as an amount of INDY given *per iAsset* and if iAsset has several LP tokens then this `k` should be proportionally distributed between its LP tokens.

When calculating `r` we either need to change `k` to mean "INDY given per collection of LP tokens corresponding to a given iAsset" or change formula to something like this:

$r = k (\sum_l \frac{x_l}{\sum_i a_i^l} y_l )$

where
- $l$ is a collection of types of LP tokens for a given iAsset (for `iUSD` it's `{lp1, lp2}`)
- $x_l$ is a current user's staked amount of a specific LP token (for a given iAsset)
- $a^l$ is a set of specific staked LP tokens for a given iAsset for each user
- $y_l$ is a proportion of total minted amount for a LP token $l$ of a total minted amount of all LP tokens for a given iAsset (for `lp1` it's `minted_amount_lp1 / (minted_amount_lp1 + minted_amount_lp2)`)